### ISSUE TITLE
Mac install instructions are missing something

### ISSUE THREAD

**Issue by funkflute** [foosel/OctoPrint#1211]
when I get to: **sudo xcodebuild** I get:

_The directory /Users/myusername does not contain an Xcode project._

what directory should I be in?

**Comment by GitIssueBot** [comment]
Hi @funkflute,

It looks like there is some information missing from your bug report that will be needed in order to solve the problem. Please take a look at the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md) which will provide you with a template to fill out here so that your bug report is ready to be investigated (I promise I'll go away then too!).

If you did not intend to report a bug, please take special note of the title format to use as described in the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md).

I'm marking this one now as needing some more information. Please understand that if you do not provide that information within the next two weeks (until 2016-02-19 00:20) I'll close this ticket so it doesn't clutter the bug tracker. This is nothing personal, so please just be considerate and help the maintainers solve this problem quickly by following the guidelines linked above. Remember, the less time the devs have to spend running after information on tickets, the more time they have to actually solve problems and add awesome new features. Thank you!

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being, so don't expect any replies from me :) Your ticket is read by humans too, I'm just not one of them.

**Comment by foosel** [comment]
Not a bug with the software, more of a support issue, so the bug tracker is actually the wrong place for that. 

That being said, that step might be wrong. Try leaving that out, see if the rest still works. I'm not a mac person and based the wiki page on how I got it to run. I needed this step so that xcode would ask me to accept its license agreement and I did it in my home directory. Maybe you already accepted the license and so it's trying to do what it's actually supposed to do instead of showing you what it showed me. Try skipping directly to the ruby line.
